android/src/org/ppsspp/ppsspp/NativeActivity.java

```diff
@@ -813,6 +820,27 @@ protected void onResume() {
 		}
 	}
 
+	@Override
+	public void onAttachedToWindow() {
```

Comment:
Ah, this seems like also a good place to setup the system UI callback.

-[Unknown]

Reply:
Yup - would happily merge such a change.